pkg/eval/builtin_fn_fs_test.go

```diff
@@ -23,3 +32,36 @@ func TestBuiltinFnFS(t *testing.T) {
 		That(`-is-dir ~/file`).Puts(false),
 	)
 }
+
+func TestBuiltinCd(t *testing.T) {
+	tmpHome, cleanup := InTempHome()
+	defer cleanup()
+
+	mustMkdirAll("d1")
+	mustMkdirAll("d2")
+	d1Path := tmpHome + string(os.PathSeparator) + "d1"
+	d2Path := tmpHome + string(os.PathSeparator) + "d2"
+	os.Chdir(d1Path)
+	os.Setenv("HOME", d1Path)
```

Comment:
/face palm :smile: Yeah, I just copy/pasted without looking at what `InTempHome` actually did. Furthermore, the `os.Chdir` is also not needed since having it be the value of `tmpHome` is fine for my purposes.